scheduler/pkg/agent/repository/mlserver/mlserver.go

```diff
@@ -321,3 +323,12 @@ func (m *MLServerRepositoryHandler) findHighestVersionInPath(modelPath string) (
 	}
 	return "", nil
 }
+
+func (m *MLServerRepositoryHandler) GetModelConfig(path string) (*agent.ModelConfig, error) {
```

Comment:
```suggestion
func (m *MLServerRepositoryHandler) GetModelConfig(_ string) (*agent.ModelConfig, error) {
```

Given that we have the argument name ("path") in the interface (for documentation purposes), and MLServer doesn't use the path at all, perhaps it's cleaner to mark that explicitly as unused? Just a suggestion, I don't have a strong view on this.